.eslintignore

```diff
@@ -0,0 +1,2 @@
+# include
+!.eleventy.js
```

Comment:
why so?

Reply:
ESLint ignores dot files by default. The `.eleventy.js` file is one of the most important JS files in the stack and should be included in the "lintables" list.

Reply:
oh sorry i misread. thought this excluded eleventy. good to go.